.gitignore

```diff
@@ -17,3 +17,7 @@ build/
 
 # Code editor settings
 .vscode
+   
+# artifact
```

Comment:
[note to self] This is the same as in [rdk's .gitignore](https://github.com/viamrobotics/rdk/blob/fcae3e91d937d54cf84bd1ee80374b09898cb7f5/.gitignore#L60-L62)